src/App.tsx

```diff
@@ -1,97 +1,80 @@
-import React from 'react';
-import './App.css';
-import { getNumbers } from './utils';
+import React, { useState } from 'react';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
+
+const totalItems = 42;
+const showOptions = [3, 5, 10, 20];
 
 export const App: React.FC = () => {
+  const searchParam = window.location.search;
+  const castomPage = +searchParam[searchParam.indexOf('=') + 1] || 1;
```

Comment:
Fix typo (fix all cases please)

```suggestion
  const customPage = +searchParam[searchParam.indexOf('=') + 1] || 1;
```